docs/pages/access-controls/sso/oidc.mdx

```diff
@@ -1,30 +1,27 @@
 ---
-title: OAuth2 and OIDC authentication for SSH
-description: How to configure SSH access with OAuth2 or OIDC (OpenID connect) using Teleport
-h1: OAuth2 / OIDC Authentication for SSH
+title: OAuth2 and OIDC authentication
+description: How to configure Teleport access with OAuth2 or OpenID connect (OIDC)
 ---
 
-This guide will explain how to configure an SSO provider using [OpenID Connect](http://openid.net/connect/)
-(also known as OIDC) to issue SSH credentials to a specific groups of users.
-When used in combination with role based access control (RBAC) it allows SSH
-administrators to define policies like:
+This guide will explain how to configure an SSO provider using [OpenID
+Connect](http://openid.net/connect/) (also known as OIDC) to issue Teleport
+credentials to a specific groups of users. When used in combination with role
+based access control (RBAC) it allows Teleport administrators to define
+policies like:
 
-- Only members of "DBA" group can SSH into machines running PostgreSQL.
+- Only members of the "DBA" group can connect to PostgreSQL databases.
 - Developers must never SSH into production servers.
 
 ## Prerequisites
 
 - Admin access to the SSO/IdP being integrated with users assigned to groups/roles.
-- Teleport role with access to maintaining `oidc` resources. This is available in the default `editor` role.
+- Teleport role with access to maintaining `oidc` resources. This is available
+  in the default `editor` role.
```

Comment:
```suggestion
- Teleport role with permission to maintain `oidc` resources. This permission is available
  in the default `editor` role.
```
The "access to maintaining" sounds a little odd to my ear. I would guess it is more like a role granted x permissions.